server/src/dtos/asset-media-response.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsOptional } from 'class-validator';
+
+export enum AssetMediaStatusEnum {
+  UPDATED = 'updated',
+  DUPLICATE = 'duplicate',
+}
+
+export class DefaultAssetMediaResponseDto {
+  @ApiProperty({
+    type: String,
+    enum: AssetMediaStatusEnum,
+    required: true,
+    enumName: 'AssetMediaStatus',
+  })
+  readonly status?: string;
+  @IsOptional()
```

Comment:
You need to use `Optional` instead of `IsOptional` because the latter has some unintended behavior.